platform_it/pom.xml

```diff
@@ -535,6 +536,26 @@
                             </includes>
                         </configuration>
                     </execution>
+                    <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = Integration tests: ROLLBACK -->
+                    <execution>
+                        <id>integration-tests-rollback</id>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
+                        </goals>
+                        <configuration>
+                            <groups>com.splicemachine.test.RollbackToTransactionTest</groups>
+                            <argLine>-Xmx2g -Xms64m</argLine>
+                            <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                            <includes>
+                                <include>**/*RollbackToTransactionIT.java</include>
```

Comment:
i think you forgot to add the file RollbackToTransactionIT

Reply:
The IT is in the EE project because the procedure requires enterprise access.  It's at splice_backup/src/test/java/com/splicemachine/backup/RollbackToTransactionIT.java

I will add the description.